pages/index.js

```diff
@@ -14,41 +14,43 @@ const generateWords = (words) => {
   return pickedWords;
 };
 
-const useStateWithLocalStorage = (localStorageKey, defaultValue) => {
-  const [value, setValue] = React.useState(() => {
-    if (typeof window !== "undefined") {
-      const localStorageValue = JSON.parse(
-        localStorage.getItem(localStorageKey)
-      );
-      return localStorageValue != null ? localStorageValue : defaultValue;
-    }
-    return defaultValue;
-  });
+const getState = (localStorageKey, defaultValue) => {
+  const localStorageValue = JSON.parse(localStorage.getItem(localStorageKey));
+  return localStorageValue != null ? localStorageValue : defaultValue;
+};
 
-  React.useEffect(() => {
-    if (typeof window !== "undefined") {
-      localStorage.setItem(localStorageKey, JSON.stringify(value));
-    }
-  }, [value instanceof Array ? value.length : value]);
+const Index = () => {
+  const [isLoading, setIsLoading] = React.useState(true);
```

Comment:
You can just import `useState` from react and the code will be less verbose. Simply
`const [isLoading, setIsLoading] = useState(true);`